src/components/form/Field.svelte

```diff
@@ -86,9 +86,11 @@
     dispatch('blur', { valid });
   }
 
-  function onInput(event: InputEvent) {
+  async function onInput(event: InputEvent) {
```

Comment:
The only thing that worries me about this change is if `Field` is used on input that might require more complex validation. For example if Field is used to validate an email, while typing the email this `input` event will fire and make the box immediately red before the user is finished typing.

Reply:
I think a `change` event throws on blur and Enter press. Could we potentially just replace the `onKeyDown` with an `onChange`?

Reply:
Ah yeah that might work! Change also fires for `select` so it would take care of that case as well

Reply:
Ah right, forgot that this is bound to inputs as well as selects. So I should modify all these keydowns to be changes? And modify all instances of keydown?

Reply:
Also one thing I'm noticing – change only fires once after the input is dirty as opposed to keydown which will fire every time. For form fields this shouldn't matter right?

Reply:
Gave it a go, see what you think.